.github/workflows/deploy.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           if [[ ${{ matrix.many-linux }} != *64 ]]; then PRE_CMD=linux32; fi
           #py=$(echo ${{ matrix.python-version }} | tr -d .)
           plat=manylinux${{ matrix.many-linux }}
-          $PRE_CMD /ws/tools/build_wheels.sh /ws ${plat} /opt/python/cp3{11,10,9,8,7}*/bin
+          $PRE_CMD /ws/tools/build_wheels.sh /ws ${plat} /opt/python/cp3{11,10,9,8}*/bin
```

Comment:
We should probably keep compatibility with python 3.7.

Debian Buster is supported until June 30th, 2024 and is release with python 3.7
If it's not too hard. https://wiki.debian.org/LTS

Reply:
No, sorry, the rule is to have 4 supported versions of Python, that takes enough of my time. Debian people will have to stay on 2.2, if they are clever enough to install their own version of rdiff-backup, they will probably be clever enough to install their own version of Python, else they go with what's delivered with their platform.